x-pack/plugin/esql/qa/testFixtures/src/main/resources/inlinestats.csv-spec

```diff
@@ -51,8 +51,35 @@ emp_no:integer | languages:integer | max_lang:integer | gender:keyword
          10014 |                 5 | 5                | null
 ;
 
+maxOfIntByKeywordLimit1Aggd
+required_capability: inlinestats_v11
+
+FROM employees
+| KEEP emp_no, languages, gender
+| INLINESTATS max_lang = MAX(languages) BY gender
+| LIMIT 1
```

Comment:
Limit 1 is non-deterministic and this might fail on multi-node tests like for serverless, no?

Maybe `LIMIT 3 | SORT gender` (-> F, M, null) as this would still surface the bug?

Reply:
The first (`emp_no`-sorted) value has a `language` value of `2`. If the LIMIT (of `1`) is pushed down, that would yield a `max_lang` of `2`. Ideally we'd have a `| SORT emp_no` before INLINESTATS, but that doesn't current work.
In any case, even if the data is shuffled, all records should have the same `max_lang` (`5`).
I've left a comment on the test.